local/docker/wordpress/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM wordpress:php7.3-apache
+ARG PHP_VERSION=7.4
+
+FROM wordpress:php${PHP_VERSION}-apache
+
+ARG XDEBUG_VERSION=2.9.6
```

Comment:
PHP 5.6 needs an earlier version so we provide an `ARG` to override it as needed on Travis.